src/Mod/Draft/draftguitools/gui_edit_draft_objects.py

```diff
@@ -117,15 +117,12 @@ def get_edit_obj_context_menu(self, edit_command, obj, position):
 
     def handle_delete_point(self, edit_command, obj, node_idx):
         self.delete_point(obj, node_idx)
-        edit_command.resetTrackers(obj)
```

Comment:
Note that all these `handle_foo` methods are now just calling a single method, so I would suggest removing them entirely and just calling that method (`delete_point` here) directly from the lambda. e.g.:

```py
        return [
                  ("delete point", lambda: self.delete_point(obj, node_idx)),
        ]
```

Reply:
done, in the second last commit. It's not so clean to introduce those methods and then remove them, I know, but but... pleeeeease, ignore it!